mobile/lib/pages/recordings_page.dart

```diff
@@ -36,25 +36,57 @@ class RecordingsPage extends HookWidget {
     final recordings = await fetchData(pageUrl, RecordingInfo.fromJson);
 
     final pageData = <RecordingInfo>[];
-    for (final element in recordings) {
-      final (recordingPath, duration) = await fetchSingle(
-        _singleRecordingUrl(element.recordingPath),
-        (jsonData) => (
-          jsonData['source_url'] as String,
-          Duration(seconds: jsonData['media_details']['length'] as int)
-        ),
-      );
 
+    final recordingDetails = {
+      for (final RecordingInfo recording in recordings)
+        recording.title: RecordingDetails(),
+    };
+    // using title as key isn't ideal to put it mildly, but out of all the
```

Comment:
I suppose that asks for an `id` field to be added

Reply:
this is a change that should affect the whole project not just the release, so I fixed it here: https://github.com/RadioAktywne/ra-app/pull/98